src/reactionsystem.jl

```diff
@@ -617,70 +628,77 @@ function ReactionSystem(eqs, iv, unknowns, ps;
                         continuous_events = nothing,
                         discrete_events = nothing,
                         metadata = nothing)
-                        
-    name === nothing &&
+               
+    # Error checks
+    if name === nothing &&
         throw(ArgumentError("The `name` keyword must be provided. Please consider using the `@named` macro"))
+    end
```

Comment:
I find these 
```julia
condition &&
    do something
```
that the formater have created confusing to parse. I tried to change them to either be a `if` statement, or be on a single line.